.eleventy.js

```diff
@@ -0,0 +1,49 @@
+const govukEleventyPlugin = require('@x-govuk/govuk-eleventy-plugin')
+
+module.exports = function(eleventyConfig) {
+
+  // Register the plugin
+  eleventyConfig.addPlugin(govukEleventyPlugin, {
+    brandColour: '#28a',
+    fontFamily: 'system-ui, sans-serif',
+    icons: {
+      mask: 'https://raw.githubusercontent.com/x-govuk/logo/main/images/x-govuk-mask-icon.svg?raw=true',
+      shortcut: 'https://raw.githubusercontent.com/x-govuk/logo/main/images/x-govuk-favicon.ico',
+      touch: 'https://raw.githubusercontent.com/x-govuk/logo/main/images/x-govuk-apple-touch-icon.png'
+    },
+    url: process.env.GITHUB_ACTIONS
+      ? 'https://x-govuk.github.io/govuk-rspec-helpers/'
+      : '/',
+    header: {
+      organisationLogo: 'x-govuk',
+      organisationName: 'X-GOVUK'
+    },
+    footer: {
+      contentLicence: {
+        html: 'Licensed under the <a class="govuk-footer__link" href="https://github.com/x-govuk/govuk-prototype-components/blob/main/LICENSE.txt">MIT Licence</a>, except where otherwise stated'
+      },
+      copyright: {
+        text: '© X-GOVUK'
+      },
+      meta: {
+        items: []
+      }
```

Comment:
Don’t need this property if empty.

```suggestion
      meta: {
        items: []
      }
```